Makefile

```diff
@@ -24,17 +24,16 @@ license:
 	@echo "License: Running licence checks.."
 	@scripts/check_license.sh
 
-linter: gotools build
-	# TODO enable linter again
-	#@echo "LINT: Running code checks for Go files..."
-	#@cd $$(/bin/pwd) && ./scripts/golinter.sh
-	#@echo "LINT: Running code checks for Cpp/header files..."
-	#@cd $$(/bin/pwd) && ./scripts/cpplinter.sh
-	#@echo "LINT completed."
+linter: gotools
```

Comment:
hmm, why did you remove the dependency on build?  Now a `make linter` will fail unless you have done a build ...

Reply:
Removed build dep for linter target as build is invoked by travis anyway and when using linter locally forcing a complete build just for running linting I feel is not good. I would argue when a developer wants to run linting, they should be able to run it with the current code without build components.

Reply:
This would be fine if target would run successfully on non-build code. However it fails!

Reply:
added build dependency back. Still I think it is not useful and it's ok that it fails when a developer does not run make build before.